code/datums/stars.dm

```diff
@@ -172,13 +172,13 @@
 				nav_icon_name = "icy"
 				icon_layers += "p_icy"
 
-			if(101 to 130)
+			if(51 to 100)
```

Comment:
The random number goes up to 130 not 100. Also this sneakily changes planet ratios in the galaxy to massively increase habitable planets spawning.

Ratio before:
5:5:3

Ratio after
1:1:2